docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -59,6 +60,7 @@ export const FrameworkChooser = ({ platform }) => {
       <ToggleButton
         value="angular"
         size="small"
+        title="Angular"
```

Comment:
I know `titles` aren't accessible, but since we already have a `VisuallyHidden` text, that should be good for screen readers! 👍